azurelinuxagent/common/logcollector.py

```diff
@@ -43,6 +43,11 @@
 OUTPUT_RESULTS_FILE_PATH = os.path.join(_LOG_COLLECTOR_DIR, "results.txt")
 COMPRESSED_ARCHIVE_PATH = os.path.join(_LOG_COLLECTOR_DIR, "logs.zip")
 
+CGROUPS_SLICE = "azure.slice"
```

Comment:
you should use _AZURE_SLICE (from cgroupconfigurator.py) instead of redefining this slice

Reply:
(or, I have another comment in your PR) pass the slice name as a command-line parameter